include/klee/Internal/Module/KModule.h

```diff
@@ -107,14 +109,30 @@ namespace klee {
     void addInternalFunction(const char* functionName);
 
   public:
-    KModule(llvm::Module *_module);
-    ~KModule();
+    KModule() = default;
 
-    /// Initialize local data structures.
+    /// Optimise and prepare module such that KLEE can execute it
     //
     // FIXME: ihandler should not be here
-    void prepare(const Interpreter::ModuleOptions &opts, 
-                 InterpreterHandler *ihandler);
+    void optimiseAndPrepare(const Interpreter::ModuleOptions &opts,
+                            InterpreterHandler *ihandler,
```

Comment:
The implementation of `optimiseAndPrepare` does not seem to use `ihandler`, so I think you can implement the `FIXME` now.

Reply:
@jbuening Good point!